--- configure  2021-10-07 13:24:51.000000000 +0000
+++ configure.new  2021-10-07 13:25:41.478415954 +0000
@@ -13662,55 +13662,55 @@



-# Define if X509_TEA_set_state exists
-
-    { $as_echo "$as_me:${as_lineno-$LINENO}: checking whether X509_TEA_set_state runs" >&5
-$as_echo_n "checking whether X509_TEA_set_state runs... " >&6; }
-    save_LIBS="$LIBS"
-    save_LDFLAGS="$LDFLAGS"
-    save_CPPFLAGS="$CPPFLAGS"
-    LDFLAGS="$LDFLAGS $OPENSSL_LDFLAGS"
-    LIBS="$OPENSSL_LIBS $LIBS"
-    CPPFLAGS="$OPENSSL_INCLUDES $CPPFLAGS"
-    if test "$cross_compiling" = yes; then :
-  { { $as_echo "$as_me:${as_lineno-$LINENO}: error: in \`$ac_pwd':" >&5
-$as_echo "$as_me: error: in \`$ac_pwd':" >&2;}
-as_fn_error $? "cannot run test program while cross compiling
-See \`config.log' for more details" "$LINENO" 5; }
-else
-  cat confdefs.h - <<_ACEOF >conftest.$ac_ext
-/* end confdefs.h.  */
-void X509_TEA_set_state(int change);
-int
-main ()
-{
-X509_TEA_set_state(0);
-  ;
-  return 0;
-}
-_ACEOF
-if ac_fn_c_try_run "$LINENO"; then :
-
-            { $as_echo "$as_me:${as_lineno-$LINENO}: result: yes" >&5
-$as_echo "yes" >&6; }
-
-$as_echo "#define HAVE_X509_TEA_SET_STATE 1" >>confdefs.h
-
-
-else
-
-            { $as_echo "$as_me:${as_lineno-$LINENO}: result: no" >&5
-$as_echo "no" >&6; }
-
-
-fi
-rm -f core *.core core.conftest.* gmon.out bb.out conftest$ac_exeext \
-  conftest.$ac_objext conftest.beam conftest.$ac_ext
-fi
-
-    CPPFLAGS="$save_CPPFLAGS"
-    LDFLAGS="$save_LDFLAGS"
-    LIBS="$save_LIBS"
+## Define if X509_TEA_set_state exists
+#
+#    { $as_echo "$as_me:${as_lineno-$LINENO}: checking whether X509_TEA_set_state runs" >&5
+#$as_echo_n "checking whether X509_TEA_set_state runs... " >&6; }
+#    save_LIBS="$LIBS"
+#    save_LDFLAGS="$LDFLAGS"
+#    save_CPPFLAGS="$CPPFLAGS"
+#    LDFLAGS="$LDFLAGS $OPENSSL_LDFLAGS"
+#    LIBS="$OPENSSL_LIBS $LIBS"
+#    CPPFLAGS="$OPENSSL_INCLUDES $CPPFLAGS"
+#    if test "$cross_compiling" = yes; then :
+#  { { $as_echo "$as_me:${as_lineno-$LINENO}: error: in \`$ac_pwd':" >&5
+#$as_echo "$as_me: error: in \`$ac_pwd':" >&2;}
+#as_fn_error $? "cannot run test program while cross compiling
+#See \`config.log' for more details" "$LINENO" 5; }
+#else
+#  cat confdefs.h - <<_ACEOF >conftest.$ac_ext
+#/* end confdefs.h.  */
+#void X509_TEA_set_state(int change);
+#int
+#main ()
+#{
+#X509_TEA_set_state(0);
+#  ;
+#  return 0;
+#}
+#_ACEOF
+#if ac_fn_c_try_run "$LINENO"; then :
+#
+#            { $as_echo "$as_me:${as_lineno-$LINENO}: result: yes" >&5
+#$as_echo "yes" >&6; }
+#
+#$as_echo "#define HAVE_X509_TEA_SET_STATE 1" >>confdefs.h
+#
+#
+#else
+#
+#            { $as_echo "$as_me:${as_lineno-$LINENO}: result: no" >&5
+#$as_echo "no" >&6; }
+#
+#
+#fi
+#rm -f core *.core core.conftest.* gmon.out bb.out conftest$ac_exeext \
+#  conftest.$ac_objext conftest.beam conftest.$ac_ext
+#fi
+#
+#    CPPFLAGS="$save_CPPFLAGS"
+#    LDFLAGS="$save_LDFLAGS"
+#    LIBS="$save_LIBS"

 # Default PAM install dir
 case "$host" in
